### PR TITLE
Add specs files to build Wazuh 4.4.0

### PR DESCRIPTION
**Pull request by wazuhci**
This PR adds the specs files to build Wazuh 4.4.0

### CODE REVIEW

File: rpms/SPECS/4.4.0/wazuh-manager-4.4.0.spec [hunk 1]
```diff
@@ -824,6 +824,8 @@ rm -fr %{buildroot}
 
 
 %changelog
+* Sat Dec 25 2021 support <info@wazuh.com> - 4.4.0
+- More info: https://documentation.wazuh.com/current/release-notes/
 * Mon Nov 01 2021 support <info@wazuh.com> - 4.3.0
 - More info: https://documentation.wazuh.com/current/release-notes/
 * Wed Oct 06 2021 support <info@wazuh.com> - 4.2.2
```
Comment (okynos) [inline rpms/SPECS/4.4.0/wazuh-manager-4.4.0.spec:831]: Update with 4.2.3, 4.2.4 and 4.2.5 changes

File: rpms/SPECS/4.4.0/wazuh-agent-4.4.0.spec [hunk 2]
```diff
@@ -612,6 +612,8 @@ rm -fr %{buildroot}
 
 
 %changelog
+* Sat Dec 25 2021 support <info@wazuh.com> - 4.4.0
+- More info: https://documentation.wazuh.com/current/release-notes/
 * Mon Nov 01 2021 support <info@wazuh.com> - 4.3.0
 - More info: https://documentation.wazuh.com/current/release-notes/
 * Wed Oct 06 2021 support <info@wazuh.com> - 4.2.2
```
Comment (okynos) [inline rpms/SPECS/4.4.0/wazuh-agent-4.4.0.spec:619]: Same here

File: aix/SPECS/4.4.0/wazuh-agent-4.4.0-aix.spec [hunk 3]
```diff
@@ -290,6 +290,8 @@ rm -fr %{buildroot}
 %attr(750, root, wazuh) %{_localstatedir}/wodles/*
 
 %changelog
+* Sat Dec 25 2021 support <info@wazuh.com> - 4.4.0
+- More info: https://documentation.wazuh.com/current/release-notes/
 * Mon Nov 01 2021 support <info@wazuh.com> - 4.3.0
 - More info: https://documentation.wazuh.com/current/release-notes/
 * Wed Oct 06 2021 support <info@wazuh.com> - 4.2.2
```
Comment (okynos) [inline aix/SPECS/4.4.0/wazuh-agent-4.4.0-aix.spec:297]: Same

File: unattended_scripts/open-distro/unattended-installation/distributed/wazuh-server-installation.sh [hunk 4]
```diff
@@ -11,7 +11,8 @@
 ## Check if system is based on yum or apt-get
 char="."
 debug='> /dev/null 2>&1'
-WAZUH_VER="4.2.2"
+WAZUH_MAJOR="4.3"
```
Comment (okynos) [inline unattended_scripts/open-distro/unattended-installation/distributed/wazuh-server-installation.sh:14]: This have to keeps stable version.

**Review by okynos — CHANGES_REQUESTED** [review]
Minor changes.

File: unattended_scripts/open-distro/unattended-installation/distributed/wazuh-server-installation.sh (hunk 4 above)
Comment (DFolchA) [reply unattended_scripts/open-distro/unattended-installation/distributed/wazuh-server-installation.sh]: Updated to 4.4, this should have the current version

File: aix/SPECS/4.4.0/wazuh-agent-4.4.0-aix.spec (hunk 3 above)
Comment (DFolchA) [reply aix/SPECS/4.4.0/wazuh-agent-4.4.0-aix.spec:303]: Done in b45ec09

File: rpms/SPECS/4.4.0/wazuh-agent-4.4.0.spec (hunk 2 above)
Comment (DFolchA) [reply rpms/SPECS/4.4.0/wazuh-agent-4.4.0.spec:625]: Done in b45ec09

File: debs/SPECS/4.4.0/wazuh-agent/debian/changelog [hunk 5]
```diff
@@ -1,8 +1,33 @@
...
+
+  * More info: https://documentation.wazuh.com/current/release-notes/
+
+ -- Wazuh, Inc <info@wazuh.com>  Thu, 21 Oct 2021 15:57:51 +0000
+
+wazuh-agent (4.2.3-RELEASE) stable; urgency=low
+
+  * More info: https://documentation.wazuh.com/current/release-notes/
+
+ -- Wazuh, Inc <info@wazuh.com>  Wed, 06 Oct 2021 15:07:13 +0000
```
Comment (okynos) [inline debs/SPECS/4.4.0/wazuh-agent/debian/changelog:27]: Typo, copied twice.

File: unattended_scripts/elastic-stack/unattended-installation/all-in-one-installation.sh [hunk 6]
```diff
@@ -8,6 +8,12 @@
 # License (version 2) as published by the FSF - Free Software
 # Foundation.
 
+WAZUH_VER="4.3.0"
```
Comment (okynos) [inline unattended_scripts/elastic-stack/unattended-installation/all-in-one-installation.sh:11]: Set it to 4.4

File: unattended_scripts/elastic-stack/unattended-installation/distributed/elastic-stack-installation.sh [hunk 7]
```diff
@@ -8,6 +8,13 @@
 # License (version 2) as published by the FSF - Free Software
 # Foundation.
 
+WAZUH_MAJOR="4.3"
+WAZUH_VER="4.3.0"
```
Comment (okynos) [inline unattended_scripts/elastic-stack/unattended-installation/distributed/elastic-stack-installation.sh:12]: Same

File: unattended_scripts/elastic-stack/unattended-installation/distributed/wazuh-server-installation.sh [hunk 8]
```diff
@@ -283,8 +283,8 @@ installFilebeat() {
         logger -e "Filebeat installation failed"
         exit 1;
     else
-        eval "curl -so /etc/filebeat/filebeat.yml https://packages.wazuh.com/resources/4.2/elastic-stack/unattended-installation/distributed/templates/filebeat.yml --max-time 300  $debug"
-        eval "curl -so /etc/filebeat/wazuh-template.json https://raw.githubusercontent.com/wazuh/wazuh/4.2/extensions/elasticsearch/7.x/wazuh-template.json --max-time 300 $debug"
+        eval "curl -so /etc/filebeat/filebeat.yml https://packages.wazuh.com/resources/4.3/elastic-stack/unattended-installation/distributed/templates/filebeat.yml --max-time 300  $debug"
```
Comment (okynos) [inline unattended_scripts/elastic-stack/unattended-installation/distributed/wazuh-server-installation.sh:286]: 4.4

File: unattended_scripts/open-distro/tools/wazuh-offline-download.sh [hunk 9]
```diff
@@ -9,8 +9,8 @@
 # Foundation.
 
 WAZUH_MAJOR="4.x"
-WAZUH_MINOR="4.2"
-DEFAULT_WAZUH_VERSION="4.2.3"
+WAZUH_MINOR="4.3"
```
Comment (okynos) [inline unattended_scripts/open-distro/tools/wazuh-offline-download.sh:12]: 4.4

File: unattended_scripts/open-distro/unattended-installation/distributed/elastic-stack-installation.sh [hunk 10]
```diff
@@ -8,14 +8,17 @@
 # License (version 2) as published by the FSF - Free Software
 # Foundation.
 
-## Check if system is based on yum or apt-get
 char="."
 debug='> /dev/null 2>&1'
-WAZUH_VER="4.2.2"
+WAZUH_VER="4.3.0"
+WAZUH_MAJOR="4.3"
```
Comment (okynos) [inline unattended_scripts/open-distro/unattended-installation/distributed/elastic-stack-installation.sh:14]: 4.4

**Review by okynos — CHANGES_REQUESTED** [review]
Minor changes.

File: CHANGELOG.md [hunk 11]
```diff
@@ -4,11 +4,26 @@ All notable changes to this project will be documented in this file.
 
 ## [v4.3.0]
```
Comment (alberpilot) [inline CHANGELOG.md:5]: Change it to 4.4.0

File: unattended_scripts/open-distro/tools/wazuh-passwords-tool.sh [hunk 12]
```diff
@@ -165,15 +166,16 @@ readFileUsers() {
...
-        logger -e "the password file doesn't have a correct format.
-        It must have this format:
-        User:
-        name: wazuh
-        password: wazuhpasword
-        User:
-        name: kibanaserver
-        password: kibanaserverpassword"
-        exit 1
+	logger -e "The password file doesn't have a correct format.
```
Comment (alberpilot) [inline unattended_scripts/open-distro/tools/wazuh-passwords-tool.sh:169]: Make a variable with formatted text and pass it to the logger.

**Review by alberpilot — CHANGES_REQUESTED** [review]
Some minor changes

File: unattended_scripts/open-distro/tools/wazuh-passwords-tool.sh (hunk 12 above)
Comment (DFolchA) [reply unattended_scripts/open-distro/tools/wazuh-passwords-tool.sh]: Done in a7004d7

File: unattended_scripts/open-distro/unattended-installation/distributed/elastic-stack-installation.sh (hunk 10 above)
Comment (DFolchA) [reply unattended_scripts/open-distro/unattended-installation/distributed/elastic-stack-installation.sh]: Done in e861bf7

File: unattended_scripts/open-distro/tools/wazuh-offline-download.sh (hunk 9 above)
Comment (DFolchA) [reply unattended_scripts/open-distro/tools/wazuh-offline-download.sh]: Done in e861bf7

File: unattended_scripts/elastic-stack/unattended-installation/distributed/wazuh-server-installation.sh (hunk 8 above)
Comment (DFolchA) [reply unattended_scripts/elastic-stack/unattended-installation/distributed/wazuh-server-installation.sh]: Done in e861bf7

File: unattended_scripts/elastic-stack/unattended-installation/distributed/elastic-stack-installation.sh (hunk 7 above)
Comment (DFolchA) [reply unattended_scripts/elastic-stack/unattended-installation/distributed/elastic-stack-installation.sh]: Done in e861bf7

File: unattended_scripts/elastic-stack/unattended-installation/all-in-one-installation.sh (hunk 6 above)
Comment (DFolchA) [reply unattended_scripts/elastic-stack/unattended-installation/all-in-one-installation.sh]: Done in e861bf7

File: debs/SPECS/4.4.0/wazuh-agent/debian/changelog (hunk 5 above)
Comment (DFolchA) [reply debs/SPECS/4.4.0/wazuh-agent/debian/changelog]: Done in 22c881b

**Review by alberpilot — APPROVED** [review]
LGTM

**Review by okynos — APPROVED** [review]
LGTM.

**Pull request merged**